src/components/MapControls.svelte

```diff
@@ -129,6 +164,18 @@
         </button>
       </div>
     </div>
+    <div class="control-section">
+      <button
+        class="fullscreen"
+        on:click={handleHideUi}
+        title={$showDisplaysStore ? 'Hide UI' : 'Show UI'}
+        style={$showDisplaysStore
+          ? ''
+          : 'position: absolute; top: 12px; right: 12px; margin-top: -1em;'}
+      >
+        <Fa icon={$showDisplaysStore ? faExpand : faCompress} />
+      </button>
+    </div>
```

Comment:
I'm tempted to have two separate buttons here, one that only appears in the controls bar and one that only appears when in fullscreen mode. That would simplify some of this since we'd just hide the entire controls bar rather than hiding each section. What do you think?